applications/machine_learning/configuration/thingy53_nrf5340_cpuapp/dts.overlay

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
Why is this file needed? My intention was that this overlay should be applied automatically by `nrf/modules/mcuboot/CMakeLists.txt` when it was needed.

Reply:
The overlay is currently applied to mcuboot child image only. We need to manually apply it to application in order to use it. With current change proposal from @tejlmand, we may consider adding the overlay next to the flash_nop source file (since these two are tightly coupled together)

Reply:
I will try to add proper modification.

Reply:
I already verified this one. We will add the dts overlay in the same way as we do it now.